root/views/layout.sml

```diff
@@ -7,17 +7,17 @@ html
       meta(http-equiv='X-UA-Compatible' content='IE=edge, chrome=1')
       meta(name='description' content='')
       meta(name='author' content='<%= github_username %>')
-      //- uncomment if your site is responsive!
-      //- meta(name="viewport" content="width=device-width, initial-scale=1")
+      meta(name="viewport" content="width=device-width, initial-scale=1")
 
     block(name='title')
       title Title of Your Site
 
     block(name='stylesheets')
       link(rel='stylesheet' href='css/index.css')
 
-  body
-    block(name='content')
+  body(role='document')
+    main(role='main')
```

Comment:
Each page **should** have a `main` element, as per [accessibility guidelines](http://accessibility.parseapp.com/#11).

The user can create their own block if they wanted to have a main header:

```
block(name="banner")
```

And using it:

```
extends(src="banner")
header(role="banner")
    h1 content here
```

So regardless, it seems we are adding some repitition.  If I were to add another `header(role="banner")` in another file, I would have to repeat the same thing.  But I feel like since each page will **alway** have a `<main>` tag, the former approach makes more sense for those starting new projects.